ajax/libs/bricklayer/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "bricklayer",
+  "filename": "bricklayer.min.js",
+  "version": "0.2.22",
+  "description": "Lightweight cascading grid layout library",
+  "homepage": "http://ademilter.github.io/bricklayer/",
+  "keywords": [
+    "masonry",
+    "bricklayer",
+    "columns",
+    "grid",
+    "responsive"
+  ],
+  "demo": "http://ademilter.github.io/bricklayer/",
+  "npmName": "bricklayer",
+  "npmFileMap": [
+    {
+      "basePath": "dist/",
+      "files": [
+        "*"
```

Comment:
Oops, a mistake here, I noticed that there is a file called `bricklayer-node.js`, is it for nodejs only? If so, we should use `"**/*!(node)"` here.